index.html

```diff
@@ -285,22 +297,43 @@ <h3 class="heading-tertiary">Skills</h3>
             </div>
           </li>
         </ul>
+
+        <!-- Absolute Positions -->
+        <img
+          src="imgs/desktop-version/illustrations/about-top-left.png"
+          alt="Beautiful illustration"
+          class="desktop-only position-absolute position--top position--left"
+        />
+        <img
+          src="imgs/desktop-version/illustrations/about-top-right.png"
+          alt="Beautiful illustration"
+          class="desktop-only position-absolute position--top position--right"
+        />
       </section>
 
       <!----------------------->
       <!-- CONTACT ME SECTION -->
       <!----------------------->
```

Comment:
On the contact Section, please make sure to follow the layout of elements from the Figma design according to [this requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md#:~:text=You%20must%20replicate%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20info%20from%20the%20Figma%20templates.):

1. According to [this requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md#:~:text=You%20must%20replicate%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20info%20from%20the%20Figma%20templates.), Kindly make sure to respect the margins between the form and the footer.
2. Kindly make sure to add the line before the Footer

**Your Design**
![Screenshot from 2023-05-04 20-07-41](https://user-images.githubusercontent.com/97834160/236296424-c61012d1-5e67-418f-bd21-12fca3a79b33.png)

**Figma Design**
![Screenshot from 2023-05-04 20-07-24](https://user-images.githubusercontent.com/97834160/236296496-32ce0658-e6f8-460e-ad3a-a26ca0586f16.png)